common/utils_unittest.cpp

```diff
@@ -131,9 +131,9 @@ UTILS_TEST(guessResolutionOverflow) {
     EXPECT_EQ(h, 0);
 
     sstream.str("");
-    sstream << long(std::numeric_limits<int>::max()) * 2 + 3
+    sstream << long(std::numeric_limits<int>::max()) * 2ll + 3
             << "x"
-            << long(std::numeric_limits<int>::max()) * 3 + 2;
+            << long(std::numeric_limits<int>::max()) * 3ll + 2;
```

Comment:
Lines 122 thru 127 should change `std::numeric_limits<long>::max()` to `std::numeric_limits<long long>::max()`